app/models/contact/project.rb

```diff
@@ -4,4 +4,10 @@ def self.policy_class
   end
 
   belongs_to :project, optional: true
+
+  def establishment_main_contact
+    return false unless project_id
+    project = ::Project.find(project_id)
```

Comment:
This is interesting! I was expecting to be able to just call `project` (the association) on the contact, but that doesn't work - not sure that is right - but not something to fix here, I will make a note to look at the Contact associations.

Reply:
It doesn't work (I think) because the object itself is a `Project` (a Contact::Project)

Reply:
Oh sorry, was typing too early without coffee 🤦  I'm not sure why just calling `project` doesn't work - maybe bc it's optional? There is something odd in these associations tbh.

Reply:
No problem! I've made a note here to have a look: https://dfe-gov-uk.visualstudio.com/Academies-and-Free-Schools-SIP/_backlogs/backlog/Complete%20conversions%20transfers%20and%20changes/Epics/?workitem=139133